internal/rpc/user_device_data_api.go

```diff
@@ -85,6 +87,26 @@ func (s *userDeviceData) GetUserDeviceData(ctx context.Context, req *pb.UserDevi
 	}, nil
 }
 
+func (s *userDeviceData) GetRawDeviceData(ctx context.Context, req *pb.RawDeviceDataRequest) (*pb.RawDeviceDataResponse, error) {
+	//Create gRPC connection
+	conn, err := grpc.Dial("address:port", grpc.WithInsecure())
+	if err != nil {
+		log.Fatalf("failed to connect to data retrieval service: %v", err)
+	}
+	defer conn.Close()
+	// Create gRPC client
+	client := pb.NewUserDeviceDataServiceClient(conn)
```

Comment:
ah so I think we've had some confusion as to what this method should do, this code is more like what you'd have in the client that calls this (eg. dimo-admin) to get the device data for a given userDeviceID. 
Will make some time today to pair on this.